app/src/main/java/de/westnordost/streetcomplete/quests/bike_parking_cover/AddBikeParkingCover.java

```diff
@@ -0,0 +1,53 @@
+package de.westnordost.streetcomplete.quests.bike_parking_cover;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddBikeParkingCover extends SimpleOverpassQuestType
+{
+	@Inject public AddBikeParkingCover(OverpassMapDataDao overpassServer)
+	{
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes, ways with amenity=bicycle_parking and access!=private and !covered and bicycle_parking!=shed and bicycle_parking!=lockers and bicycle_parking!=building";
```

Comment:
> `bicycle_parking!=shed and bicycle_parking!=lockers and bicycle_parking!=building`

This can be made shorthand to `bicycle_parking !~ shed|lockers|building`. It is also faster.